uiowa/uiowa
# Linkit Webform Matcher Support

Resolves https://github.com/uiowa/uiowa/issues/1442

This relies on an uncommitted patch to Linkit.

# Story

As a web editor, I want to easily link to a webform on my site using the internal link capabilities to prevent the absolute url from breaking.

# To Test

- sync any site (assumes the default webform "contact" has not be deleted from the site)
- Add or edit content with a WYSIWYG area. Click the link button and start typing "contact." If the webform exists, it should show as a result under a webform heading.
- select the result and save the page. the page should link to the contact form.
- try with other forms on the site or create new forms.
